cime_config/machines/config_machines.xml

```diff
@@ -1269,23 +1269,25 @@
         <command name="load">cmake/3.21.3</command>
         <command name="load">cray-hdf5-parallel/1.12.2.1</command>
         <command name="load">cray-netcdf-hdf5parallel/4.9.0.1</command>
-	<command name="load">cray-parallel-netcdf/1.12.3.1</command>
+        <command name="load">cray-parallel-netcdf/1.12.3.1</command>
         <command name="unload">darshan-runtime</command>
       </modules>
     </module_system>
-    
+
     <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
     <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
     <TEST_TPUT_TOLERANCE>0.1</TEST_TPUT_TOLERANCE>
     <MAX_GB_OLD_TEST_DATA>0</MAX_GB_OLD_TEST_DATA>
     <environment_variables>
       <env name="NETCDF_PATH">$ENV{NETCDF_DIR}</env>
       <env name="PNETCDF_PATH">$ENV{PNETCDF_DIR}</env>
+      <env name="HDF5_ROOT"/> <!-- frontier cmake cannot find_package(HDF5) for some reason, so disable it -->
```

Comment:
How does *xml work with cmake? Naively, I would think setting HDF5_ROOT here is the way to point the build to hdf5 in case find_package() fails, but it is actually not the case?

Reply:
Note that the text section for this element is blank, this tells CIME to UNSET this env var. When HDF5_ROOT is not set, our cmake will not try to find_package it.